misc/openapi-ballerina/modules/openapi-to-ballerina-generator/src/main/java/org/ballerinalang/openapi/typemodel/BallerinaOpenApiParameter.java

```diff
@@ -0,0 +1,43 @@
+package org.ballerinalang.openapi.typemodel;
```

Comment:
Missing license header. Check all the new classes

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/20891